docs/source/release_notes/release-notes.rst

```diff
@@ -44,75 +42,86 @@ develop
+           This should reduce the size of our transitive dependencies, and therefore the size of product binaries.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1578>`__)
 
     *    - |devbreak|
-         - Modified the type signature of `BatchingVisitableView#of` to no long accept `final BatchingVisitable<? extends T> underlyingVisitable` and instead accept
-           `final BatchingVisitable<T> underlyingVisitable`.  We do not believe this typing is in use.  If you discover that is not the case and you cannot work around it,
-           please file a ticket on the AtlasDB github page.
+         - Modified the type signature of ``BatchingVisitableView#of`` to no longer accept ``final BatchingVisitable<? extends T> underlyingVisitable`` and instead accept ``final BatchingVisitable<T> underlyingVisitable``.
+           This will resolve an issue where newer versions of Intellij fail to compile AtlasDB.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1582>`__)
 
     *    - |improved|
          - Reduced logging noise from large Cassandra gets and puts by removing ERROR messages and only providing stacktraces at DEBUG.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1590>`__)
 
     *    - |new|
-         - Upon startup of a Cassandra-backed AtlasDB client with a :ref:`Timelock block <timelock-client-configuration>`, the client will now automatically migrate its timestamp to the Timelock cluster.
+         - Upon startup of an AtlasDB client with a ``timeblock`` :ref:`config block <timelock-client-configuration>`, the client will now automatically migrate its timestamp to the the :ref:`external Timelock cluster <external-timelock-service>`.
+
            The client will fast-forward the Timelock Server's timestamp bound to that of the embedded service.
            The client will now also *invalidate* the embedded service's bound, backing this up in a separate row in the timestamp table.
-           This is not cause for concern if you have already done a manual migration to Timelock, as fast forwarding to an earlier bound is a no-op.
 
-           So far, automated migration is only supported for Cassandra KVS.
+           Automated migration is only supported for Cassandra KVS at the moment.
            If using DBKVS or other key-value services, it remains the user's responsibility to ensure that they have performed the migration detailed in :ref:`Migration to External Timelock Services <timelock-migration>`.
            (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1569>`__,
-           `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1570>`__ and
-           `Pull Request 3 <https://github.com/palantir/atlasdb/pull/1579>`__)
+            `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1570>`__, and
+            `Pull Request 3 <https://github.com/palantir/atlasdb/pull/1579>`__)
```

Comment:
Sphinx's "checkstyle" will reject this (unexpected indentation), unfortunately - it sees a bracket as a text character.